docker/base/Dockerfile

```
@@ -33,6 +33,9 @@ ARG BUILD_GIT_REV
 USER root
 COPY . /vt/src/vitess.io/vitess
 
+# Trust dubious ownership of git directory
+RUN git config --global --add safe.directory /vt/src/vitess.io/vitess
+
```

Alternatively, you could specify the owner on the `COPY` command like this:

```suggestion
COPY --chown=root . /vt/src/vitess.io/vitess
```

---

I was wondering why we don't run into this problem with `vitess/lite` and these comments explain why.
`vitess/lite` has the following
```
# Re-copy sources from working tree.
COPY --chown=vitess:vitess . /vt/src/vitess.io/vitess
```
So another +1 to this recommended solution.

---

Thanks all for this recommendation @mattlord @deepthi @arthurschreiber ❤️ I pushed a fix in fc7d57e110.